### PR TITLE
IN-141: Added standard labels to all yaml files.

### PR DESCRIPTION
**Pull request by michaelamiethyst**

### CODE REVIEW

File: charts/fluent-bit/templates/configmap.yaml [hunk 1]
```diff
@@ -7,6 +7,13 @@ metadata:
     chart: {{ template "namewithversion" . }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/fluent-bit/templates/configmap.yaml]: use _helper template namewithversion

File: charts/fluent-bit/templates/daemonset.yaml [hunk 2]
```diff
@@ -11,6 +11,13 @@ metadata:
     chart: "{{template "namewithversion" .}}"
     release: "{{.Release.Name}}"
     heritage: "{{.Release.Service}}"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/fluent-bit/templates/daemonset.yaml]: above

File: charts/fluent-bit/templates/rbac.yaml [hunk 3]
```diff
@@ -10,6 +10,13 @@ metadata:
     chart: "{{template "namewithversion" .}}"
     release: "{{.Release.Name}}"
     heritage: "{{.Release.Service}}"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/fluent-bit/templates/rbac.yaml]: above

File: charts/fluent-bit/templates/service-account.yaml [hunk 4]
```diff
@@ -10,3 +10,10 @@ metadata:
     chart: "{{template "namewithversion" .}}"
     release: "{{.Release.Name}}"
     heritage: "{{.Release.Service}}"
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: infrastructure
+    app.kubernetes.io/part-of: logging-central
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```
Comment (mikeln) [inline charts/fluent-bit/templates/service-account.yaml]: above

**Comment by mikeln** [comment]
_helpers  needs to be fixed for:

```
{{- define "namewithversion" -}}
{{- $name := default .Chart.Name .Values.nameOverride -}}
{{- printf "%s-%s" $name .Chart.Version | replace "+" "_" | trunc 63  -}}
{{- end -}}
```

**Comment by michaelamiethyst** [comment]
I've made all requested changes.  Thank you.

**Pull request merged**